docs/units/combining_and_defining.rst

```diff
@@ -50,6 +50,13 @@ complex operations. To do this, it is recommended to use
 
 Defining units
 ==============
+Users are free to define new units, either fundamental or compound
+using the `~astropy.units.def_unit` function.
+
+.. _defining_units:
+
+Example
+=======
```

Comment:
If you wish this example to have its own header it should be a subsection in the Defining Units section and not a separate section.